### ISSUE TITLE
Issues with sliders training

### ISSUE THREAD

**Issue by davizca** [rohitgandikota/sliders#39]
Hi!

Im trying to run the training but I have this problem:
![a0dbd658802179b6b73627f702a9252d](https://github.com/rohitgandikota/sliders/assets/118778759/5b202a45-a9ce-470c-9fd9-a7ef1d5cf485)

Do I have to modify something in config.xml?
[
![86f1f021590ca1cc66a51434f7ca0a45](https://github.com/rohitgandikota/sliders/assets/118778759/ebd90fb9-e74e-4be5-9773-ade366e8732a)
](url)

It's weird cause I make it working in other PC, Im running windows. Could you help me? Thanks!

**Comment by rohitgandikota** [comment]
I think it has something to do with the custom pretrained model. But if you are saying this works in other PC, then I would suggest installing the dependencies as given in our README. Also - some windows related training issues are addressed here https://github.com/rohitgandikota/sliders/issues/27#issuecomment-1833572579

**Comment by davizca** [comment]
> I think it has something to do with the custom pretrained model. But if you are saying this works in other PC, then I would suggest installing the dependencies as given in our README. Also - some windows related training issues are addressed here [#27 (comment)](https://github.com/rohitgandikota/sliders/issues/27#issuecomment-1833572579)

You were right, it was the omegaconf thing. Now it's trying to start training but gives me an error. I think I split bad the foldering.

This is my main folder (sliders folder with repo):
![main folder](https://github.com/rohitgandikota/sliders/assets/118778759/f5cb0ffc-c0a9-444c-8049-73d7b96b9332)

In Attributes I have the 2 subfolders with 5 images in each one 
![subfolders 2](https://github.com/rohitgandikota/sliders/assets/118778759/fd55c7d2-d1ef-4cfe-a7d4-710240fa782f)

I don't understand the error of "verylow" but seems related totally to my folders.
![Error](https://github.com/rohitgandikota/sliders/assets/118778759/1b24b038-e0d9-497e-b007-d6f033c8ca32)

I'm 100% sure this is my fault but I think I did the foldering right?

Thanks in advance!

**Comment by rohitgandikota** [comment]
Awesome! To run visual sliders, README is a good resource.

during the training script you need to add more arguments to specify your folder structure

`--folder_main 'D:\\sliders\\attributes\\' --folders 'high, low' --scales '-1, 1' `  add this to your  training script and it should work.

**Comment by davizca** [comment]
Working. Tried changing guidance scale, learning rate, attn and steps. Sales 10 and -10. Working at least the training, thanks a lot. We will see how it turns out. You can close this. Thanks.
